### PR TITLE
Add v1.5.2, v1.6.0 and v1.7.0 to CHANGELOG.md

### PR DESCRIPTION
**Pull request by spencerhance**

### CODE REVIEW

**Comment by rramkumar1** [comment]
@spencerhance I think this is missing some PR's? 

We had a lot of stuff go into 1.6

**Comment by spencerhance** [comment]
@rramkumar1 Updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,199 @@
...
+- Update docs for contributing  [\#697](https://github.com/kubernetes/ingress-gce/pull/697) ([rramkumar1](https://github.com/rramkumar1))
+- e2e StatusManager init\(\) handle existing config map [\#696](https://github.com/kubernetes/ingress-gce/pull/696) ([spencerhance](https://github.com/spencerhance))
+- Completely remove EnableBackendConfig flag [\#695](https://github.com/kubernetes/ingress-gce/pull/695) ([rramkumar1](https://github.com/rramkumar1))
+- TestBasicEdge -\> TestEdge [\#694](https://github.com/kubernetes/ingress-gce/pull/694) ([rramkumar1](https://github.com/rramkumar1))
+- Add e2e test for Ingress that has long namespace+name [\#689](https://github.com/kubernetes/ingress-gce/pull/689) ([rramkumar1](https://github.com/rramkumar1))
+- Port basic NEG test from k/k [\#679](https://github.com/kubernetes/ingress-gce/pull/679) ([spencerhance](https://github.com/spencerhance))
+- Add tests for HTTPS  [\#671](https://github.com/kubernetes/ingress-gce/pull/671) ([rramkumar1](https://github.com/rramkumar1))
+- Move tests for app protocols from k/k to Ingress-GCE repo [\#668](https://github.com/kubernetes/ingress-gce/pull/668) ([rramkumar1](https://github.com/rramkumar1))
```
Comment (rramkumar1) [inline CHANGELOG.md]: How come some PR's got skipped? Like 670 for example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -756,4 +950,10 @@
 
 
 
+\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```
Comment (rramkumar1) [inline CHANGELOG.md]: Can remove the excess

File: CHANGELOG.md (hunk 1 above)
Comment (spencerhance) [reply CHANGELOG.md]: Turned out most of those PRs were extraneous, fixed now

File: CHANGELOG.md (hunk 2 above)
Comment (spencerhance) [reply CHANGELOG.md]: Done

**Comment by spencerhance** [comment]
@rramkumar1 updated

File: CHANGELOG.md (hunk 1 above)
Comment (rramkumar1) [reply CHANGELOG.md]: What do you mean by extraneous? I still don't see 670.

Comment (spencerhance) [reply CHANGELOG.md]: Before, there were a ton of PRs under 1.5.2 that weren't actually added in that version, so I removed them. 

I believe the numbers being skipped refer to closed PRs and open issues

This 670: https://github.com/kubernetes/ingress-gce/pulls/670

**Pull request merged**